scripts/metrics-md.py

```diff
@@ -0,0 +1,101 @@
+import json
```

Comment:
* move this into script/utils
* please document how to use that
* include generated MD file(s) - you can put then under cmd/jaeger/docs/migration